lib/widgets/app_header.dart

```diff
@@ -229,6 +247,50 @@ class AppHeader extends StatelessWidget {
 		);
 	}
 
+	Widget buildWidget(BuildContext context) {
+		return buildHeaderContainer(context,
+			Column(
+				children: <Widget>[
+					Row(
+						mainAxisAlignment: MainAxisAlignment.spaceBetween,
+						crossAxisAlignment: CrossAxisAlignment.center,
+						children: <Widget>[
+							Column(
+							  children: <Widget>[
+							    Row(
+							      children: <Widget>[
+							        BackIconButton(),
+											Text(
+												AppLocales.of(context).translate(title),
+												style: Theme.of(context).textTheme.headline1.copyWith(color: Colors.white)
+											)
+							      ],
+							    )
+								],
+							),
+							Column(
+							  children: <Widget>[
+							  	// TODO: change this widget to widget made by Miko
+							    Padding(
+							      padding: const EdgeInsets.symmetric(horizontal: 8.0),
+							      child: IconButton(icon: Icon(Icons.help, color: Colors.white, size: 26,), onPressed: null,),
+							    ),
+							  ],
+							)
+						]
+					),
```

Comment:
Those Columns & Row are redundant as thay have only one child / children could be merged into the Row higher up
```suggestion
							BackIconButton(),
							Text(
								AppLocales.of(context).translate(title),
								style: Theme.of(context).textTheme.headline1.copyWith(color: Colors.white)
							),
							Padding(
					      padding: const EdgeInsets.symmetric(horizontal: 8.0),
					      child: IconButton(icon: Icon(Icons.help, color: Colors.white, size: 26,), onPressed: null,),
					    ),
						]
					),
```